tests/app/notifications/test_validators.py

```diff
@@ -197,6 +208,26 @@ def test_check_template_is_for_notification_type_pass(template_type, notificatio
     )
 
 
+@pytest.mark.parametrize(
+    "notification_type, expected_value", [(SMS_TYPE, 50_000), (LETTER_TYPE, 50_000), (EMAIL_TYPE, 50_000)]
+)
+def test_get_daily_rate_limit_value_for_test_keys_for_live_services(sample_service, notification_type, expected_value):
+    sample_service.email_message_limit = 50_000
+    sample_service.sms_message_limit = 50_000
+    sample_service.letter_message_limit = 50_000
```

Comment:
If you make the value different it’s a stronger assertion that it’s looking up the right one:

```suggestion
@pytest.mark.parametrize(
    "notification_type, expected_value", [(EMAIL_TYPE, 111), (SMS_TYPE, 222), (LETTER_TYPE, 333)]
)
def test_get_daily_rate_limit_value_for_test_keys_for_live_services(sample_service, notification_type, expected_value):
    sample_service.email_message_limit = 111
    sample_service.sms_message_limit = 222
    sample_service.letter_message_limit = 333
```